packages/rollup-core/src/app/data/data-service.ts

```diff
@@ -576,25 +557,26 @@ export class DefaultDataService implements DataService {
    * @inheritDoc
    */
   public async getVerificationCandidate(): Promise<VerificationCandidate> {
```

Comment:
would `getNextVerificationCandidate` be accurate since this always returns a non-array?

Reply:
Hmm... I suppose at any given time, by definition, there is only one verification candidate, so I don't know that next is required, but I'm happy to add it!